assets/js/src/core/modules/element/dynamic-types/defintinitions/objects/data-related/types/abstract/dynamic-type-object-data-abstract-input.tsx

```diff
@@ -26,10 +26,12 @@ export abstract class DynamicTypeObjectDataAbstractInput extends DynamicTypeObje
     return (
       <Input
         autoComplete="off"
+        className={ props.className }
```

Comment:
Not sure where the className prop here is initially defined? In general those class names should be defined more in the wrapper of the concret component (as discussed based on the `getObjectDataFormItemProps` method of the dynamic type class.